Another wasm2c issue in fuzz_opt

seed: `17863318942536380310`
```c++
wasm2c b.wasm -o wasm.c
clang main.c wasm.c /.../wabt/wasm2c/wasm-rt-impl.c -I/.../binaryen/third_party/wabt/wasm2c -lm -Werror
main.c:51:3: error: implicit declaration of function 'init' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  init();
  ^
main.c:60:7: error: use of undeclared identifier 'Z_hangLimitInitializerZ_vv'; did you mean 'Z_bZ_hangLimitInitializer'?
    (*Z_hangLimitInitializerZ_vv)();
      ^~~~~~~~~~~~~~~~~~~~~~~~~~
      Z_bZ_hangLimitInitializer
./wasm.h:99:15: note: 'Z_bZ_hangLimitInitializer' declared here
extern void (*Z_bZ_hangLimitInitializer)(void);
              ^
main.c:70:69: error: use of undeclared identifier 'Z_hashMemoryZ_iv'
            printf("[fuzz-exec] note result: hashMemory => %d\n", (*Z_hashMemoryZ_iv)());
                                                                    ^
main.c:74:15: error: use of undeclared identifier 'Z_funcZ_vjidjffd'
            (*Z_funcZ_vjidjffd)(0, 0, 0, 0, 0, 0, 0);
              ^
main.c:78:15: error: use of undeclared identifier 'Z_func_invokerZ_vv'
            (*Z_func_invokerZ_vv)();
              ^
main.c:82:15: error: use of undeclared identifier 'Z_func_0_invokerZ_vv'; did you mean 'Z_bZ_func_0_invoker'?
            (*Z_func_0_invokerZ_vv)();
              ^~~~~~~~~~~~~~~~~~~~
              Z_bZ_func_0_invoker
./wasm.h:49:15: note: 'Z_bZ_func_0_invoker' declared here
extern void (*Z_bZ_func_0_invoker)(void);
              ^
main.c:86:15: error: use of undeclared identifier 'Z_func_1_invokerZ_vv'; did you mean 'Z_bZ_func_1_invoker'?
            (*Z_func_1_invokerZ_vv)();
              ^~~~~~~~~~~~~~~~~~~~
              Z_bZ_func_1_invoker
./wasm.h:51:15: note: 'Z_bZ_func_1_invoker' declared here
extern void (*Z_bZ_func_1_invoker)(void);
              ^
main.c:90:68: error: use of undeclared identifier 'Z_func_3Z_fv'
            printf("[fuzz-exec] note result: func_3 => %.17e\n", (*Z_func_3Z_fv)());
                                                                   ^
main.c:94:65: error: use of undeclared identifier 'Z_func_4Z_if'
            printf("[fuzz-exec] note result: func_4 => %d\n", (*Z_func_4Z_if)(0));
                                                                ^
main.c:98:15: error: use of undeclared identifier 'Z_func_4_invokerZ_vv'; did you mean 'Z_bZ_func_4_invoker'?
            (*Z_func_4_invokerZ_vv)();
              ^~~~~~~~~~~~~~~~~~~~
              Z_bZ_func_4_invoker
./wasm.h:57:15: note: 'Z_bZ_func_4_invoker' declared here
extern void (*Z_bZ_func_4_invoker)(void);
              ^
main.c:102:15: error: use of undeclared identifier 'Z_func_5_invokerZ_vv'; did you mean 'Z_bZ_func_5_invoker'?
            (*Z_func_5_invokerZ_vv)();
              ^~~~~~~~~~~~~~~~~~~~
              Z_bZ_func_5_invoker
./wasm.h:59:15: note: 'Z_bZ_func_5_invoker' declared here
extern void (*Z_bZ_func_5_invoker)(void);
              ^
main.c:106:68: error: use of undeclared identifier 'Z_func_6Z_fv'
            printf("[fuzz-exec] note result: func_6 => %.17e\n", (*Z_func_6Z_fv)());
                                                                   ^
main.c:110:15: error: use of undeclared identifier 'Z_func_6_invokerZ_vv'; did you mean 'Z_bZ_func_6_invoker'?
            (*Z_func_6_invokerZ_vv)();
              ^~~~~~~~~~~~~~~~~~~~
              Z_bZ_func_6_invoker
./wasm.h:63:15: note: 'Z_bZ_func_6_invoker' declared here
extern void (*Z_bZ_func_6_invoker)(void);
              ^
main.c:114:15: error: use of undeclared identifier 'Z_func_7Z_vv'
            (*Z_func_7Z_vv)();
              ^
main.c:118:83: error: use of undeclared identifier 'Z_func_8Z_jv'
            printf("[fuzz-exec] note result: func_8 => %" PRId64 "\n", (int64_t)(*Z_func_8Z_jv)());
                                                                                  ^
main.c:122:15: error: use of undeclared identifier 'Z_func_8_invokerZ_vv'; did you mean 'Z_bZ_func_8_invoker'?
            (*Z_func_8_invokerZ_vv)();
              ^~~~~~~~~~~~~~~~~~~~
              Z_bZ_func_8_invoker
./wasm.h:69:15: note: 'Z_bZ_func_8_invoker' declared here
extern void (*Z_bZ_func_8_invoker)(void);
              ^
main.c:126:65: error: use of undeclared identifier 'Z_func_9Z_ifi'
            printf("[fuzz-exec] note result: func_9 => %d\n", (*Z_func_9Z_ifi)(0, 0));
                                                                ^
main.c:130:15: error: use of undeclared identifier 'Z_func_10_invokerZ_vv'; did you mean 'Z_bZ_func_10_invoker'?
            (*Z_func_10_invokerZ_vv)();
              ^~~~~~~~~~~~~~~~~~~~~
              Z_bZ_func_10_invoker
./wasm.h:73:15: note: 'Z_bZ_func_10_invoker' declared here
extern void (*Z_bZ_func_10_invoker)(void);
              ^
main.c:134:84: error: use of undeclared identifier 'Z_func_11Z_jv'
            printf("[fuzz-exec] note result: func_11 => %" PRId64 "\n", (int64_t)(*Z_func_11Z_jv)());
                                                                                   ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
wasm.c:94:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i32_load, u32, u32, u32)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:95:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i64_load, u64, u64, u64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:96:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(f32_load, f32, f32, f32)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:97:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(f64_load, f64, f64, f64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:98:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i32_load8_s, s8, s32, u32)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:99:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i64_load8_s, s8, s64, u64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:100:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i32_load8_u, u8, u32, u32)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:101:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i64_load8_u, u8, u64, u64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:102:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i32_load16_s, s16, s32, u32)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:103:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i64_load16_s, s16, s64, u64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:104:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i32_load16_u, u16, u32, u32)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:105:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i64_load16_u, u16, u64, u64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:106:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i64_load32_s, s32, s64, u64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:107:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_LOAD(i64_load32_u, u32, u64, u64)
^
wasm.c:82:5: note: expanded from macro 'DEFINE_LOAD'
    wasm_rt_memcpy(&result, &mem->data[addr], sizeof(t1)); \
    ^
wasm.c:108:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_STORE(i32_store, u32, u32)
^
wasm.c:90:5: note: expanded from macro 'DEFINE_STORE'
    wasm_rt_memcpy(&mem->data[addr], &wrapped, sizeof(t1));            \
    ^
wasm.c:109:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_STORE(i64_store, u64, u64)
^
wasm.c:90:5: note: expanded from macro 'DEFINE_STORE'
    wasm_rt_memcpy(&mem->data[addr], &wrapped, sizeof(t1));            \
    ^
wasm.c:110:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_STORE(f32_store, f32, f32)
^
wasm.c:90:5: note: expanded from macro 'DEFINE_STORE'
    wasm_rt_memcpy(&mem->data[addr], &wrapped, sizeof(t1));            \
    ^
wasm.c:111:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_STORE(f64_store, f64, f64)
^
wasm.c:90:5: note: expanded from macro 'DEFINE_STORE'
    wasm_rt_memcpy(&mem->data[addr], &wrapped, sizeof(t1));            \
    ^
wasm.c:112:1: error: implicit declaration of function 'wasm_rt_memcpy' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
DEFINE_STORE(i32_store8, u8, u32)
^
wasm.c:90:5: note: expanded from macro 'DEFINE_STORE'
    wasm_rt_memcpy(&mem->data[addr], &wrapped, sizeof(t1));            \
    ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
!
-----------------------------------------
Exception:
  File "./scripts/fuzz_opt.py", line 1322, in <module>
  File "./scripts/fuzz_opt.py", line 1118, in test_one
  File "./scripts/fuzz_opt.py", line 775, in handle_pair
  File "./scripts/fuzz_opt.py", line 784, in run_vms
  File "./scripts/fuzz_opt.py", line 707, in run
  File "./scripts/fuzz_opt.py", line 80, in run
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
-----------------------------------------
!
1
['clang', 'main.c', 'wasm.c', '.../binaryen/third_party/wabt/wasm2c/wasm-rt-impl.c', '-I/.../binaryen/third_party/wabt/wasm2c', '-lm', '-Werror']
================================================================================
You found a bug! Please report it with

  seed: 17863318942536380310
```

--- a/scripts/fuzz_opt.py
+++ b/scripts/fuzz_opt.py
@@ -767,8 +767,9 @@ class CompareVMs(TestCaseHandler):
                     D8(),
                     D8Liftoff(),
                     D8TurboFan(),
-                    Wasm2C(),
-                    Wasm2C2Wasm()]
+                    # Wasm2C(),
+                    # Wasm2C2Wasm()
+                    ]
 
     def handle_pair(self, input, before_wasm, after_wasm, opts):
         before = self.run_vms(before_wasm)

--- a/scripts/fuzz_opt.py
+++ b/scripts/fuzz_opt.py
@@ -767,6 +767,7 @@ class CompareVMs(TestCaseHandler):
                     D8(),
                     D8Liftoff(),
                     D8TurboFan(),
+                    # FIXME: Temprorary disable. See issue #4741 for more details
                     # Wasm2C(),
                     # Wasm2C2Wasm()
                     ]

